### PR TITLE
providers/aws: aws_elb: Allow alnum plus hyphen in name attribute

### PR DESCRIPTION
**Pull request by pmoust**
@radeksimko : https://github.com/hashicorp/terraform/pull/2517#issuecomment-117362827

### CODE REVIEW

**Comment by radeksimko** [comment]
Ah, that has been overlooked, sorry about that!

File: builtin/providers/aws/resource_aws_elb.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ func resourceAwsElb() *schema.Resource {
 				ForceNew: true,
 				ValidateFunc: func(v interface{}, k string) (ws []string, errors []error) {
 					value := v.(string)
-					if !regexp.MustCompile(`^[0-9a-z-]+$`).MatchString(value) {
+					if !regexp.MustCompile(`^[0-9A-Za-z-]+$`).MatchString(value) {
 						errors = append(errors, fmt.Errorf(
 							"only lowercase alphanumeric characters and hyphens allowed in %q", k))
```
Comment (radeksimko) [inline builtin/providers/aws/resource_aws_elb.go]: can you please update the error message accordingly? Then it's good to go.

Comment (pmoust) [inline builtin/providers/aws/resource_aws_elb.go]: :+1:

**Comment by radeksimko** [comment]
Thanks a lot Panagiotis and really sorry for any troubles caused by this bug.

**Pull request merged**